profiler/src/ProfilerEngine/Datadog.Linux.ApiWrapper/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Project definition
 # ******************************************************
 
-project("Datadog.Linux.ApiWrapper" VERSION 1.0.0)
+project("Datadog.Linux.ApiWrapper" VERSION 2.16.0)
```

Comment:
You'll need to make sure this is set to the correct version at time of merge (2.18.0, probably) 🙂

Reply:
Oh now it has changed :)

Reply:
✔️